torch/_inductor/codecache.py

```diff
@@ -527,7 +527,7 @@ def _reduce_tensor(
         # TODO: These tensors don't currently pickle, so we can't cache a
         # compiled graph containing them. Just fail now. If mkldnn tensors
         # get pickling support, we can remove this.
-        raise BypassFxGraphCache
+        raise BypassFxGraphCache("Bypassing cache.  mkldnn tensors unpickleable.")
```

Comment:
you dont need to write bypassing cache on each of them, that's already evident from the name of the exception

Reply:
Fixed.